backend/core/configuration.py

```diff
@@ -409,6 +433,7 @@ def translate_learn_rule_dicts_to_yaml_conf(
             :param rule_prefix: <str>
             :param rule_asns: <list><int>
             :param rules: <list><dict>
+            :param ignore_existing: <bool>
```

Comment:
@slowr any ideas on how to refactor this to become less complex? As a method I mean. I will try to move code out in smaller functions, but if it disorients from the issue we can do the refactoring another time (the important thing is that it works, but ideally I would like to also keep it simple ;) ).

Reply:
Fixed by modularizing code, now C901 does not complain.